crates/bifrost/src/loglets/local_loglet/options.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::time::Duration;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Options {
+    pub rocksdb_threads: usize,
+    pub rocksdb_disable_statistics: bool,
+    pub rocksdb_disable_wal: bool,
```

Comment:
Do we want to allow disabling the wal for the local loglet? This might be a lurking footgun?

Reply:
I wanted to leave the option available but what matters more that LogStore supports it (planning your reuse parts of it later)